docs/user-guide/user-roadmap-zowe-cli copy.md

```diff
@@ -0,0 +1,161 @@
+
+- [**Zowe CLI FAQs**](../getting-started/zowe_faq.md#zowe-cli-faq)
+
+   If you have a question, check out the FAQ, which answers the most commonly asked questions about Zowe CLI.
+
+## Quick start
+
+> Zowe skill level: Beginner
+
+- [**Zowe CLI quick start**](../getting-started/cli-getting-started.md)
+
+   Get started with Zowe CLI quickly and easily.
+
+- [**Blog: Getting Started with Zowe CLI**](https://medium.com/zowe/getting-started-with-zowe-cli-7a29420c6be7)
+
+   This blog explains how to get started with Zowe CLI quickly.
+
+## Installing
+
+> Zowe skill level: Beginner
+
+- [**System requirements**](../user-guide/systemrequirements-cli.md)
+
+   Review this topic to ensure that your system meets the requirements for installing Zowe CLI. 
+
+- [**Installing Zowe CLI**](../user-guide/cli-installcli.md)
+
+   Follow the steps to install a new release of Zowe CLI.
+
+## Configuring and updating
+
+> Zowe skill level: Intermediate
+
+- [**Configuring Zowe CLI environment variables**](../user-guide/cli-configuringcli-ev.md)
+
+   Explains how to configure Zowe CLI environment variables, such as changing log levels, setting the home directory location, and daemon mode.
+
+- [**Updating Zowe CLI**](../user-guide/cli-updatingcli.md)
+
+   Learn how to update Zowe CLI to a more recent version using different methods. 
+
+## Using Zowe CLI and plug-ins
+
+> Zowe skill level: Intermediate
+
+- [**Using Zowe CLI**](../user-guide/cli-using-usingcli.md)
+
+   Learn about how to use Zowe CLI, including connecting to the mainframe, managing profiles, integrating with API Mediation Layer, and more.
+
+- [**Zowe CLI extensions and plug-ins**](../user-guide/cli-extending.md)
+
+   This information guides you to explore the extensions and plug-ins available to Zowe CLI and install plug-ins to extend the capabilities of Zowe CLI. Plug-ins add functionality to the product in the form of new command groups, actions, objects, and options.
+
+- **Docs: Zowe CLI command reference guide**
+
+   View detailed documentation on commands, actions, and options in Zowe CLI. The reference document is based on the `@zowe-v3-lts` version of the CLI.
+
+   The content also contains the web help for all Zowe ecosystem-conformant plug-ins that contributed to this website.
+   
+   You can read an interactive online version, download a PDF document, or download a ZIP file containing the HTML for the online version:
+   - <b><a href="/stable/web_help/index.html" target="_blank">Browse online</a></b>
+   - <b><a href="/stable/CLIReference_Zowe.pdf" target="_blank">Download CLI reference in PDF format</a></b>
+   - <b><a href="/stable/zowe_web_help.zip" target="_blank">Download CLI reference in ZIP format</a></b>
+
+- **Best practices**
+
+   Are you looking for recommendations and tips on how to best use Zowe CLI to meet your needs? These resources provide best practices recommendations.
```

Comment:
Regarding "These resources", there is just one blog, so it may make sense to revise this.